etl/steps/data/garden/rff/2025-05-19/emissions_weighted_carbon_price.meta.yml

```diff
@@ -0,0 +1,147 @@
+      co2_with_ets_as_share_of_world_co2:
+        title: CO₂ emissions covered by an ETS as a share of the world's CO₂ emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_ets_as_share_of_world_ghg:
+        title: CO₂ emissions covered by an ETS as a share of the world's GHG emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_tax_as_share_of_co2:
+        title: CO₂ emissions covered by a carbon tax as a share of the country's CO₂ emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_tax_as_share_of_ghg:
+        title: CO₂ emissions covered by a carbon tax as a share of the country's GHG emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_tax_as_share_of_world_co2:
+        title: CO₂ emissions covered by a carbon tax as a share of the world's CO₂ emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_tax_as_share_of_world_ghg:
+        title: CO₂ emissions covered by a carbon tax as a share of the world's GHG emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_tax_or_ets_as_share_of_co2:
+        title: CO₂ emissions covered by a carbon tax or an ETS as a share of the country's CO₂ emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_tax_or_ets_as_share_of_ghg:
+        title: CO₂ emissions covered by a carbon tax or an ETS as a share of the country's GHG emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_tax_or_ets_as_share_of_world_co2:
+        title: CO₂ emissions covered by a carbon tax or an ETS as a share of the world's CO₂ emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      co2_with_tax_or_ets_as_share_of_world_ghg:
+        title: CO₂ emissions covered by a carbon tax or an ETS as a share of the world's GHG emissions
+        unit: '%'
+        short_unit: '%'
+        display:
+          numDecimalPlaces: 2
+      price_with_ets_weighted_by_share_of_co2:
+        title: Average price on emissions covered by an ETS, weighted by the share of the country's CO₂ emissions
+        description_short: *description_short_dollars
+        unit: constant 2019 US$ per tonne of CO₂ equivalents
+        short_unit: $/t
+        display:
+          numDecimalPlaces: 2
+          shortUnit: $
+      price_with_ets_weighted_by_share_of_ghg:
+        title: Average price on emissions covered by an ETS, weighted by the share of the country's GHG emissions
+        description_short: *description_short_dollars
+        unit: constant 2019 US$ per tonne of CO₂ equivalents
```

Comment:
Perhaps good to add the footnote in the harmonizing doc i.e: "This data is expressed in constant 2019 US$”.